code/controllers/master.dm

```diff
@@ -216,7 +216,7 @@ GLOBAL_REAL(Master, /datum/controller/master) = new
 		old_runlevel = "NULL"
 
 	testing("MC: Runlevel changed from [old_runlevel] to [new_runlevel]")
-	current_runlevel = log(2, new_runlevel) + 1
+	current_runlevel = RUNLEVEL_FLAG_TO_INDEX(new_runlevel)
```

Comment:
It's not worth making this a define, considering this is the only line where it's used and every define adds compiler overhead